src/main/ruby/truffleruby/core/truffle/concurrent_map.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved. This
+# code is released under a tri EPL/GPL/LGPL license. You can use it,
+# redistribute it and/or modify it under the terms of the:
+#
+# Eclipse Public License version 2.0, or
+# GNU General Public License version 2, or
+# GNU Lesser General Public License version 2.1.
+
+module TruffleRuby
+  class ConcurrentMap
+    def initialize(initial_capacity: nil, load_factor: nil)
```

Comment:
We can't put the actual default values here, as then we'd have to do a dance in Concurrent Ruby to filter its options hash and then splat the options. It's a bit of a mess.

Reply:
Done.